app/graphql/types/user_type.rb

```diff
@@ -2,8 +2,8 @@ module Types
   class UserType < Types::BaseObject
     field :id, ID, null: false
     field :username, String, null: true
-    field :public_debates, Types::DebateType.connection_type, null: true
-    field :public_proposals, Types::ProposalType.connection_type, null: true
-    field :public_comments, Types::CommentType.connection_type, null: true
+    field_with_deprecated_sibling :public_debates, Types::DebateType.connection_type, null: true
```

Comment:
could a line break like in the files with more fields also be helpful here and vote_type and tag_type for readability?